<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QDialog" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="down_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="over_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>温度设置大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="WindSpeed_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WorkStyle_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentTem_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UnSettedTem_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
font: 75 24pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>设定信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="high_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>高</string>
   </property>
  </widget>
  <widget class="QPushButton" name="low_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>低</string>
   </property>
  </widget>
  <widget class="QPushButton" name="report">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>报表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="windspeed_button">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>480</y>
     <width>131</width>
     <height>43</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>降低目标温度</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>风速</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="model_button">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>410</y>
     <width>131</width>
     <height>43</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="report_group">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>312</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>用量统计</string>
   </property>
   <widget class="QLabel" name="MoneyUsing_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="money_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QLabel" name="ElecUsing_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="elec_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用量</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="login_group">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>300</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>用户登录</string>
   </property>
   <widget class="QLabel" name="slave_number_label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>从机号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="slave_number_field_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(0, 0, 0);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn_2">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>109</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="model_group">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>286</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
background-image:url();
background-color: rgb(170, 0, 127, 0%);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="cool_button_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>选择制冷</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="warm_button_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>选择供暖</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="current_choice_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>当前模式：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="speed_group">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>274</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;STHeiti&quot;;
background-image:url();
background-color: rgb(85, 85, 255, 0%);</string>
   </property>
   <property name="title">
    <string>三种风速选择</string>
   </property>
   <widget class="QRadioButton" name="low_button_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>22</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>低速风</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="medium_button_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>47</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>中速风</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="high_button_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>72</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;STHeiti&quot;;</string>
    </property>
    <property name="text">
     <string>高速风</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed_button_2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>100</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
background-image: url();
background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="login_state">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>5</y>
     <width>19</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>650</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 102, 102);
background-color: rgb(102, 102, 102 ,100%);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>0</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>37</y>
     <width>650</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="table" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="again">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>325</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>设定风速：</string>
   </property>
  </widget>
  <widget class="QLabel" name="work_state_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
color: rgb(0,0,0);
font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>工作状态信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_temp_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
font: 75 24pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>设定信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>52</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
font: 13pt &quot;STHeiti&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>设定温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_state_label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>52</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
color: rgb(0, 0, 0);
font: 13pt &quot;STHeiti&quot;;</string>
   </property>
   <property name="text">
    <string>室温与风速：</string>
   </property>
  </widget>
  <widget class="QLabel" name="work_state">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
color: rgb(0,0,0);
font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>工作状态信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="master_button">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>450</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>开机</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>175</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 13pt &quot;STHeiti&quot;;</string>
   </property>
   <property name="text">
    <string>实时温度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ccc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>85</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ccc_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>85</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>table</zorder>
  <zorder>down_btn</zorder>
  <zorder>up_btn</zorder>
  <zorder>middle_btn</zorder>
  <zorder>over_btn</zorder>
  <zorder>label</zorder>
  <zorder>WindSpeed_label</zorder>
  <zorder>WorkStyle_label</zorder>
  <zorder>ok_btn</zorder>
  <zorder>stop_btn</zorder>
  <zorder>CurrentTem_label</zorder>
  <zorder>UnSettedTem_label</zorder>
  <zorder>high_btn</zorder>
  <zorder>low_btn</zorder>
  <zorder>report</zorder>
  <zorder>windspeed_button</zorder>
  <zorder>model_button</zorder>
  <zorder>report_group</zorder>
  <zorder>login_group</zorder>
  <zorder>model_group</zorder>
  <zorder>speed_group</zorder>
  <zorder>close</zorder>
  <zorder>min</zorder>
  <zorder>login_state</zorder>
  <zorder>again</zorder>
  <zorder>set_label_2</zorder>
  <zorder>work_state_2</zorder>
  <zorder>set_temp_2</zorder>
  <zorder>set_label</zorder>
  <zorder>login_state_label_2</zorder>
  <zorder>work_state</zorder>
  <zorder>master_button</zorder>
  <zorder>label_2</zorder>
  <zorder>ccc</zorder>
  <zorder>ccc_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
